bpf/bpf_lxc.c

```diff
@@ -800,9 +800,12 @@ ipv6_policy(struct __sk_buff *skb, int ifindex, __u32 src_label, int *forwarding
 			return ret2;
 	}
 
-	verdict = policy_can_access_ingress(skb, src_label, tuple.dport,
-					    tuple.nexthdr, sizeof(tuple.saddr),
-					    &tuple.saddr, false);
+	if (!(cfg->flags & EP_F_SKIP_POLICY_INGRESS))
```

Comment:
Can be added has an optimization later, @joestringer would the `unlikely` help here? (As it is more likely an endpoint to have policy enforced)

Reply:
Seems reasonable, but agree that it doesn't need to block the PR.